### ISSUE TITLE
Can not simplify (sqrt(2)+2*sqrt(x))/(1+sqrt(2)*sqrt(x))

### ISSUE THREAD

**Issue by michaeldongyuxi** [sympy/sympy#11500]
Should this equation simplified to sqrt(2)?

**Comment by justinsuen** [comment]
Yeah it does. As long as 1-2x does not equal 0. You can see this by multiplying what you have by (1 - sqrt(2)*sqrt(x))/(1 - sqrt(2)*sqrt(x)).

**Comment by deep-shekhar** [comment]
when i put x = 2, or 1/2 or 0, the output comes as sqrt(2), but giving any another value of x say, x=5,just gives the output as,
(sqrt(2) + 2*sqrt(5))/(1 + sqrt(10)).
shouldn't the output come as sqrt(2) in every case?

**Comment by ylemkimon** [comment]
You have to extend the algebraic domain to include sqrt(2). You can do this by explicitly passing the extension, i.e., `cancel(expr, extension=[sqrt(2)])`, or by `cancel(expr, extension=True)`, which automatically extends it.
However, I think `simplify` should do this.(Related #5428)
